jmxtrans-output/jmxtrans-output-elastic/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jmxtrans</groupId>
+		<artifactId>jmxtrans-output</artifactId>
+		<version>251-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>jmxtrans-output-elastic</artifactId>
+
+	<name>JmxTrans - ElasticSearch output writers</name>
+
+	<properties>
+		<verify.mutationThreshold>80</verify.mutationThreshold>
+		<verify.totalBranchRate>75</verify.totalBranchRate>
+		<verify.totalLineRate>75</verify.totalLineRate>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>io.searchbox</groupId>
+			<artifactId>jest</artifactId>
+			<version>0.1.6</version>
```

Comment:
Could you move `<version/>` to the main pom? That way we have all versions of all dependencies in a centralized place.